src/vs/workbench/contrib/chat/common/chatService.ts

```diff
@@ -305,6 +305,8 @@ export interface IChatTerminalToolInvocationData {
 	alternativeRecommendation?: string;
 	language: string;
 	terminalToolSessionId?: string;
+	terminalCommandUri?: UriComponents;
```

Comment:
Can't we just use `URI` here? `UriComponents` normally is only needed when dealing with remote/IPC.